keyboards/c39/keymaps/default/keymap.c

```diff
@@ -0,0 +1,50 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+
+#define _FN1 2
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```

Comment:
QMK defines these keycode aliases automatically. Lines 12-13 can be deleted.

https://github.com/qmk/qmk_firmware/blob/b90df560f8f5a681eac193b779cb8c5ecb20d681/quantum/quantum_keycodes.h#L29-L31

```suggestion
```